src/sd.cpp

```diff
@@ -1806,7 +1809,8 @@ int main(int argc, char** argv)
             {
                 auto vec = ::read_file<tensor_vector<float>>(g_main_args.m_decode_latents.c_str());
 
-                ncnn::Mat sample(128, 128, 4);
+                size_t latent_dim = g_main_args.m_turbo ? 64 : 128;
+                ncnn::Mat sample(latent_dim, latent_dim, 4);
```

Comment:
Small fix for `--decode-latents` aswell since the latents produced by SDXLTurbo are 64x64 rather than 128x128